src/main/app/case/formatter/uploaded-files.ts

```diff
@@ -14,17 +14,24 @@ export const fromApiApplicant1 = (data: Partial<CaseData>): Partial<Case> => ({
   coClarificationUploadedFiles:
     data.coClarificationUploadDocuments?.map(file => ({
       id: `${file.id}`,
-      name: `${file.value.documentFileName}`,
+      name: `${getFilename(file.value)}`,
     })) || [],
 });
 
 export const fromApiApplicant2 = (data: Partial<CaseData>): Partial<Case> => ({
   applicant2UploadedFiles:
     data.applicant2DocumentsUploaded?.map(file => ({
       id: `${file.id}`,
-      name: `${file.value.documentFileName}`,
+      name: `${getFilename(file.value)}`,
     })) || [],
   applicant2DocumentsUploaded: data.applicant2DocumentsUploaded,
   applicant2CannotUpload: data.applicant2CannotUploadSupportingDocument?.length ? Checkbox.Checked : Checkbox.Unchecked,
   applicant2CannotUploadDocuments: data.applicant2CannotUploadSupportingDocument,
 });
+
+export const getFilename = (document: Partial<DivorceDocument> | undefined | null): string | undefined => {
+  if (document) {
```

Comment:
you could probably get rid of this by doing:

```
return document?.documentFileName || document.documentLink?.document_filename;
```

Reply:
done